xcp/net/ifrename/__init__.py

```diff
@@ -20,3 +20,8 @@
 # ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import re
+
+# .util.py and .logic.py need to import VALID_ETH_NAME
+VALID_ETH_NAME = re.compile(r"^eth([\d])+$")
```

Comment:
What's the difference between `([\d])+` and `\d+`? I understand this is existing code.

Reply:
Yes, the way it is used in `xcp` files, it would make no difference,
but e.g. `host-installer/netutil.py` imports VALID_ETH_NAME from `xcp`
and while it also just checks if it matches or not, other users could read the group defined by the `([\d])+`, so it's safer to not change it.